codeSnippets/snippets/tutorial-kotlin-rpc-client/src/main/kotlin/Main.kt

```diff
@@ -0,0 +1,60 @@
+package com.example
+
+import com.example.model.Pizza
+import com.example.model.PizzaShop
+import io.ktor.client.*
+import io.ktor.client.plugins.websocket.*
+import io.ktor.http.*
+import kotlinx.coroutines.runBlocking
+import kotlinx.rpc.client.withService
+import kotlinx.rpc.serialization.json
+import kotlinx.rpc.transport.ktor.client.rpc
+import kotlinx.rpc.transport.ktor.client.rpcConfig
+import kotlinx.rpc.streamScoped
+
+fun main() = runBlocking {
+    val ktorClient = HttpClient {
+        install(WebSockets)
+    }
+
+    val client = ktorClient.rpc {
+        url {
+            host = "localhost"
+            port = 8080
+            encodedPath = "pizza"
+        }
+
+        rpcConfig {
+            serialization {
+                json()
+            }
+        }
+    }
+
+    val pizzaShop: PizzaShop = client.withService<PizzaShop>()
+
+/*
```

Comment:
Commented code

Reply:
The commented out parts are from the first iteration and are referenced in the tutorial. I wanted to avoid hard-coding them into the `.topic` file. Do you think it's better to create a separate file / code sample for each iteration?